src/spatial/math/mod.rs

```diff
@@ -34,22 +34,29 @@ pub fn hash_region(offset: &V3c<f32>, size: f32) -> u8 {
         + (offset.y >= half_size) as u8 * 4
 }
 
-fn bitmask_mapping_64bits(x: usize, y: usize, z: usize, size: usize) -> usize {
+/// Maps 3 dimensional space limited by `size` to 1 dimension
+/// This mapping function supposes that the coordinates are bound inside
+/// a cube, each dimension `size` long.
+/// * `x` - x coordinate of position
+/// * `y` - y coordinate of position
+/// * `z` - z coordinate of position
+/// * `size` - Range of the given coordinate space
+fn bitmask_mapping(x: usize, y: usize, z: usize, size: usize) -> usize {
     x + (y * size) + (z * size * size)
 }
 
 /// Returns with a bitmask to select the relevant octant based on the relative position
 /// and size of the covered area
 fn position_in_bitmask_64bits(x: usize, y: usize, z: usize, size: usize) -> u8 {
-    let pos_inside_bitmask = (V3c::new(x, y, z) * 4) / size;
-    debug_assert!(pos_inside_bitmask.x < 4);
-    debug_assert!(pos_inside_bitmask.y < 4);
-    debug_assert!(pos_inside_bitmask.z < 4);
-    let pos_inside_bitmask = bitmask_mapping_64bits(
-        pos_inside_bitmask.x,
-        pos_inside_bitmask.y,
-        pos_inside_bitmask.z,
-        size,
+    let pos_inside_bitmask_space = (V3c::new(x, y, z) * 4) / size;
+    debug_assert!(pos_inside_bitmask_space.x < 4);
+    debug_assert!(pos_inside_bitmask_space.y < 4);
+    debug_assert!(pos_inside_bitmask_space.z < 4);
+    let pos_inside_bitmask = bitmask_mapping(
+        pos_inside_bitmask_space.x,
+        pos_inside_bitmask_space.y,
+        pos_inside_bitmask_space.z,
+        4,
```

Comment:
4 because that's the brick size? maybe we should make that a constant then, so it's clear why it's 4?

Reply:
it's 4 because the resolution one can store in 64 bits is 4. 

8x8x8 resolution would be 512 bit, which could be stored in 16 u32, or 8 u64 variables ( WGPU only supports u32 ), for optimization purposes 4x4x4 seems like a good balance. 

It can also be contained in the current SizedNode data, so no need to extend the structure.